protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -146,6 +146,33 @@ An existing LP has `average entry valuation 1090.9` and `S=110`. Currently the s
 (average entry valuation) = 1090.9
 ```
 
+### Calculating supplied liquidity from probability of trading 
+
+At every vega time change calculate the supplied liquidity provided by each committed LP. 
+This is done by taking into account all the volume they're providing between the tightest price monitoring bound and then 
+use the formula provided by [probability weighted liquidity measure](./0034-PROB-prob_weighted_liquidity_measure.ipynb).
+
+Now calculate the total provided liquidity by committed LPs:
+```
+total_provided = sum over committed LPs of liquidity provided by a committed LP
+```
```

Comment:
Could you clarify this calculation? If it's meant to equal the total provided by *all* LPs, then why does it mention *a* committed LP at the end?

Reply:
Committed in this case just means they submitted the 0038-OLIQ liquidity order.